### DISCUSSION TITLE
Making action column with custom class name ?

### DISCUSSION THREAD

**Discussion by hadyjsc** [Okipa/laravel-table#126] [Q&A] [answered]
Hi dear, i think laravel-table has the feature to change html class name on action column, but how to use $class on RedirectRowAction constructor?  An example i try to update this func

`public function __construct(public string $showUrl, public bool $openInNewWindow = false) {}` to `public function __construct(public string $showUrl, public bool $openInNewWindow = false, public array $class) {}` and call the $class like `class: $class` in ShowRowAction class constructor. Then its well done, i have the custom class name 😁

<img width="246" alt="image" src="https://user-images.githubusercontent.com/16421715/226085092-fb773fe8-1243-4e63-8637-c08b34bb1d23.png">

**Comment by Okipa** [comment, marked as answer]
Hi @hadyjsc, sorry for the late answer.

You may simply create a new row action with `php artisan make:table:row:action RedirectRowAction` according to your needs and use this one in your table.

Check here how you can reuse `RedirectRowAction` to create your own row action: https://github.com/Okipa/laravel-table/blob/master/src/RowActions/ShowRowAction.php

Or you can directly use `RedirectRowAction` in your table if you prefer.

**Reply by hadyjsc** [reply]
Thanks @Okipa , my mistake didn't read the documentation.
